### DISCUSSION TITLE
Reset DB

### DISCUSSION THREAD

**Discussion by IanBUK** [DependencyTrack/dependency-track#1606] [Q&A] [answered]
Is there a way to reset the Dependency Track DB, so that users and permissions remain but all the projects and vulnerabilities recorded against them are wiped?

I've been through and deleted all the projects/versions and wanted the vulnerability counts to be reset too - we're still looking at how we avoid an ever growing list of projects in the console. We release to a set of customers every fortnight, a different build for each of them and if each release is shown as a different project then we'd be looking at over 390 items in the console, each year.

**Comment by stevespringett** [comment, marked as answer]
A few thousand projects is quite common. To avoid these projects being analyzed or having their metrics updated, set the projects to inactive.

Short of that, there is no way to reset a database, but the APIs can be used to achieve this. Although I would likely go straight to the database.
